change-the-middle-management-mindset.md

```diff
@@ -0,0 +1,83 @@
+## Title
+* Changing the middle-management mindset
+
+## Problem
+* Key problem: the willingness to allocate resources 
+* Embedded accountability problem: they can't account for the time they spend and put it in their objectives. Need some metric to make it clear it is worthwhile. Has to become a KPI for them.
+
+## Context
+* top down InnerSource support. Embedded in their objectives? Trickling down? Top level management has determined a new KPI around InnerSource and it is cascaded down to middle-management? No.
+    - a vacuum between top down support and the objectives for developers
+    - top down support but no one knows what that means
+* developer wants to stand up and be a part of IS projects
+* there is no incentive that fits into the middle-management objectives
+* difficult for middle-management to understand how to enable InnerSource; how to work for controlling the direct output of one team to trying to embrace and let evolve an IS community?
+
+* Developers contribute and PO/PM or scrum master finds out and puts a wall between developers and the IS project
+* Middle-management agrees but P0 or P1 gets in the way
+* They would rather duplicate than reuse and collaborate; rewriting the fast and easy way is prioritized over participating in a reusable collaboration.
+* Not yet having a majority microservices architecture is not a factor.
+* For code changes beyond bug fixes
+
+## Forces
+* Organizational goals rarely happen with Middle-Management; they write their own goals (or their bosses do); otherwise incentivization happens through budget. Centralized incentivization is very difficult (their other goals will suffer). KPIs tied to people's bottom line can be effective.
+* Educational component: propensity is to blame evil middle-management; they may not know how it works and need to understand it. What benefit they will have for having their people
+* Managers are afraid of having people stolen from them, of having priorities that aren't theirs, of becoming irrelevant.
+* Managers might fear that this will become the wild west (we're the only ones who really care)
+* How to manage priorities in such a bazaar? 
+* Middle-management lacks understanding of what IS implies
+* Middle-management has the ability to learn about IS (formalized training)
+
+## Solution
+* OKR = objective key result - bigger picture. The best tool ever to get serious traction across business organizations. We are one team; creating durable teams horizontally across BUs. Tie middle-management into the OKRs; they can tie these into their L2s they write (WAgile: quarterly we do big planning sessions, L2s are a quarterly goal).  Epic for a year, L2 for a quarter.
```

Comment:
If company is doing Agile Dev, during release planning, time & resources for IS practices should be built into sprints.

Reply:
I have added this suggestion to the solutions section.